app/src/main/java/invoice_automation/module/QuickBooksModule.java

```diff
@@ -2,18 +2,16 @@
 
 import com.intuit.ipp.core.Context;
 import com.intuit.ipp.core.ServiceType;
-import com.intuit.ipp.data.Customer;
-import com.intuit.ipp.data.EmailStatusEnum;
-import com.intuit.ipp.data.Invoice;
+import com.intuit.ipp.data.*;
```

Comment:
nit: can you follow this page to stop wildcard imports if you're using intellij: https://www.jetbrains.com/help/idea/creating-and-optimizing-imports.html#disable-wildcard-imports